### ISSUE TITLE
[Help Wanted] System Alert Wont Respond

### ISSUE THREAD

**Issue by Lakr233** [Lakr233/ActivateMac#6]
https://user-images.githubusercontent.com/25259084/170406541-681e8e54-dc5a-483d-b127-8ba3f8eae68d.mov

**Comment by Naville** [comment]
Try overriding:

```objc
NSWindow
......
- (BOOL)canBecomeKeyWindow {
        return NO;
}
- (BOOL)canBecomeMainWindow {
        return NO;
}
.....
```

**Comment by Lakr233** [comment]
> Try overriding:
> 
> ```objc
> NSWindow
> ......
> - (BOOL)canBecomeKeyWindow {
>         return NO;
> }
> - (BOOL)canBecomeMainWindow {
>         return NO;
> }
> .....
> ```

It's already there :(

<img width="504" alt="截屏2022-05-29 17 37 14" src="https://user-images.githubusercontent.com/25259084/170861632-525b9ea9-6fbd-4adc-bbd7-84348b8321c4.png">

**Comment by Naville** [comment]
Not sure where ``WindowLevel = 2005`` comes from, but according to ``<CoreGraphics/CGWindowLevel.h>``, kCGPopUpMenuWindowLevel should be 101,and 2005 is much bigger than 101

**Comment by Naville** [comment]
You're probably looking for kCGStatusWindowLevel

**Comment by Lakr233** [comment]
ref: https://github.com/Lakr233/ActivateMac/pull/8#issuecomment-1140467058

**Comment by Lakr233** [comment]
Fixed in macOS 12.5 (21G72)
